docs/general/installation/container.md

```diff
@@ -311,28 +311,29 @@ SuccessExitStatus=0 143
 WantedBy=default.target
 ```
 
-## TrueNAS SCALE / TrueCharts
+## TrueNAS SCALE
 
-Jellyfin is available as a [TrueNAS SCALE](https://www.truenas.org/) App inside the [TrueCharts](https://www.truecharts.org/) App Catalog with direct integration into the GUI, no CLI needed. Direct support is available on the [TrueCharts Discord](https://discord.gg/tVsPTHWTtr) and the source code is available on [GitHub](https://github.com/truecharts/charts).
+Jellyfin is available as a [TrueNAS SCALE](https://www.truenas.org/) App in the official app catalog with direct integration into the GUI, no CLI needed.
 
-1. Install the TrueCharts Catalog to TrueNAS SCALE, see [website](https://truecharts.org/manual/SCALE/guides/getting-started/#adding-truecharts) for more info.
+1. Go to Apps page from the top level SCALE menu
 
-   1. Go to Apps page from the top level SCALE menu
-   2. Select Manage Catalogs tab on the Apps page
-   3. Click Add Catalog
-   4. After reading the iXsystems notice, click Continue and enter the required information:
-      Name: truecharts
-      Repository: `https://github.com/truecharts/catalog`
-      Preferred Trains: `enterprise` and `stable`
-      Branch: main
-   5. Click Save and allow SCALE to refresh its catalog with TrueCharts (this may take a few minutes)
+2. Click `Discover Apps` and search for `Jellyfin`
 
-2. Click `Available Applications` and search for `Jellyfin`
+3. Click `Install`, which will take you to the GUI installation wizard and you'll be able to fill out the necessary info
 
-3. Click `Install`, which will take you to the GUI Wizard and you'll be able to fill out the necessary info
+   - Published Server URL to publish in UDP Auto Discovery response
+   - User and Group Configuration, Network Configuration
```

Comment:
Mention that 'host' mode will be required if DLNA is used

Reply:
Ohh, so that's why DLNA didn't seem to work for me xD

Reply:
it is mentioned in the docs https://jellyfin.org/docs/general/installation/container#docker

Reply:
Yeah I found it there after you mentioned it. Must have missed it otherwise, and people using Docker Compose setups are already covered since it's put there in the example config file.